tests/06-hello-font/input/amitu/index.ftd

```diff
@@ -9,4 +9,26 @@ font: $assets.fonts.Roboto
 font: $assets.fonts.myFirstFont
 
 -- ftd.text: World
-font: $theme-assets.fonts.themeFont
\ No newline at end of file
+font: $theme-assets.fonts.themeFont
+
+-- ftd.text:
+text: $assets.files.index.jpg.dark
```

Comment:
Do they also have equivalent `$assets.files.index.jpg.dark-page`?

Reply:
Other alternative is we do not have `-page` suffix and instead use `$assets.pages.index.jpg` etc.

Reply:
1. Not yet.
2. Hmm... I like the `$assets.pages` syntax better.